tests/framework/config/kind-config-buildkite.yml

```diff
@@ -14,4 +14,30 @@ nodes:
     kind: ClusterConfiguration
     apiServer:
       certSANs:
-        - "docker"
\ No newline at end of file
+        - "docker"
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+    - containerPort: 30379
```

Comment:
There is no use case to expose the GCS port, so we can remove 6379 at this moment.

Reply:
Done